ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -122,13 +129,22 @@ svi_profiles:
     enable_mlag_ibgp_peering_vrfs: < true | false >
 
     # Define L3 network services organized by vrf.
+    
+    # Define IPv6 unicast-routing for network services organized by vrf.
+    # Define for all VRFs under the tenant
+    ipv6_routing: < true | false, default -> false >
```

Comment:
couldn't we just auto-detect this? If any ipv6 stuff is configured, we add this? Ping me for implementation hints.